CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,49 @@
+# Kubeflow Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+* Examples of unacceptable behavior by participants include:
```

Comment:
Markdown formatting error

"Examples" should not be an item in the current list and the next line "The use of sexualized" probably should be a list item.

Reply:
Agreed. Looks like a copypasta error, thanks for catching!